client/packages/system/src/Encounter/ListView/columns.ts

```diff
@@ -22,26 +22,27 @@ interface useEncounterListColumnsProps {
   includePatient?: boolean;
 }
 
-export const useEncounterAdditionalInfoAccessor: ColumnDataAccessor<
-  EncounterRowFragment,
-  string[]
-> = ({ rowData }): string[] => {
+const useEncounterAdditionalInfoAccessor: () => {
+  additionalInfoAccessor: ColumnDataAccessor<EncounterRowFragment, string[]>;
+} = () => {
   const t = useTranslation();
-  const additionalInfo = [];
-
-  if (rowData?.activeProgramEvents[0]?.data) {
-    additionalInfo.push(rowData.activeProgramEvents[0].data);
-  }
+  return {
+    additionalInfoAccessor: ({ rowData }): string[] => {
+      const additionalInfo = rowData.activeProgramEvents
+        .map(e => (e.type?.startsWith('status') ? e.data : undefined))
```

Comment:
would it be worth adding a helper function for the filter & map, since it is used twice?

Reply:
Ok moved it to a utils method even though don't like that the utils methods is so far away from where it is used (defined it in the `program` package and its used in `system`)

Reply:
oh, yes. that's a reasonable concern. It does hide away the functionality when reading the code. Happy for you to ignore me in that case..